### PR TITLE
Feature/markers for places on map

### PR DESCRIPTION
**Pull request by klesogor**
For now you can simply set markers on map(but not update already set markers!). This should be implemented later on. Also we should think about custom popup and marker

### CODE REVIEW

File: frontend/src/services/map/markerGeneratorService.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                                        ${markerData.name}
+                                   </strong>
+                                </div>
+                                <div>
+                                   ${markerData.address || ''} 
+                                </div>
+                           </div>
+                    </div>`);
+    return popup;
+    };
```
Comment (nemoipaha) [inline frontend/src/services/map/markerGeneratorService.js]: Fix indentation

File: frontend/src/services/map/markerManagerService.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import MarkerGenerator from './markerGeneratorService';
+
+class MarkerManagerService {
```
Comment (nemoipaha) [inline frontend/src/services/map/markerManagerService.js]: remove console.log )

Comment (klesogor) [reply frontend/src/services/map/markerManagerService.js]: removed :sunglasses:

File: frontend/src/services/map/markerGeneratorService.js (hunk 1 above)
Comment (klesogor) [reply frontend/src/services/map/markerGeneratorService.js]: fixed it

**Pull request merged**